serokell-style.md

```diff
@@ -5,7 +5,7 @@ Serokell Haskell Style Guide
 > This style guide's aims are code beauty, readability and understandability.
 
 You can find our other formatting utilites and guidelines which expand the code style:
-* [.stylish-haskell.yaml](https://github.com/serokell/serokell-core/blob/master/.stylish-haskell.yaml) config
+* [.stylish-haskell.yaml](https://github.com/serokell/serokell-util/blob/master/.stylish-haskell.yaml)
 * [Universum](https://github.com/serokell/universum)
 * [Custom _HLint_ rules](https://github.com/input-output-hk/cardano-sl/blob/master/HLint.hs)
```

Comment:
I think it'd be good not to refer to the rules in `cardano-sl` repository, because there can be some cardano-sl-specific rules (maybe not now, but in future). We can just copy them into this repository, for example.

Reply:
I don't think it's important for now. After this issue is done, I can refer to rules in `universum`:

* https://github.com/serokell/universum/issues/98